test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesControllerSpec.scala

```diff
@@ -1419,11 +1420,19 @@ class ExemptionAndLossesControllerSpec
     checkPageIsDisplayed(
       performAction(data),
       messageFromMessageKey(pageTitleKey, titleArgs: _*), { doc =>
+      
         doc.select("#error-summary-display > ul > li > a").text() shouldBe messageFromMessageKey(
           expectedErrorMessageKey,
           errorArgs: _*
         )
+
         doc.title() should startWith("Error:")
+
+        val errorSummary     = doc.select(s"""a[href="#$key"]""").text()
+        val inlineErrorMsg  = doc.select(s"""span[id="$key-inline-error"]""").text()
+        val errorSummaryMsg = if (errorSummary.nonEmpty) "Error: " + errorSummary else errorSummary
```

Comment:
```scala
errorSummary.nonEmpty
```
should never be true right since we should always have an error summary?

Reply:
I found 4 tests were failing without that logic. Okay will check them again